tests/sanity/ignore-2.10.txt

```diff
@@ -150,6 +150,8 @@ plugins/module_utils/univention_umc.py future-import-boilerplate
 plugins/module_utils/univention_umc.py metaclass-boilerplate
 plugins/module_utils/vexata.py future-import-boilerplate
 plugins/module_utils/vexata.py metaclass-boilerplate
+plugins/action/iptables_state.py action-plugin-docs
+plugins/modules/system/iptables_state.py validate-modules:undocumented-parameter
```

Comment:
New `ignore.txt` entries are not allowed. Please fix the problems instead of ignoring them.

Reply:
the module uses internally two options (_back and _timeout) that are passed in by the action plugin, not the by user. So I'm before this choice:
- document _back and _timeout in DOCUMENTATION's options, and remove this 'ignore' case.
- do not document _back and _timeout as options, and still ignore 'validate-modules:undocumented-parameter'
So I will revert to the first alternative.

Reply:
No, please do not document internal parameters. Keep the ignore line.

Reply:
ok, that was the first idea.